src/OpenLoco/Message.h

```diff
@@ -36,10 +36,40 @@ namespace OpenLoco
         advice,
     };
 
+    enum class NewsItemSubTypes : uint8_t
```

Comment:
Probably best to stick to singular for types?

Reply:
see later